.coveragerc

```diff
@@ -1,6 +1,7 @@
 [run]
-disable_warnings = no-data-collected
+data_file = .coverage-reports/.coverage
```

Comment:
Because many files like `.coverage.{hostname}.23098.826306` will be created, I don't want to make them appear in the root directory. It is better to materialize them inside a dedicated subdirectory, which can easily be made excluded from being indexed by your favorite IDE.

With this setting, the files will be materialized like `.coverage-reports/.coverage.{hostname}.23098.826306`.